frontend/src/scenes/pipeline/ReplayMenu.tsx

```diff
@@ -0,0 +1,465 @@
+                                },
+                            ]}
+                        />
+                    )
+                },
+            }}
+            columns={[
+                {
+                    key: 'retries',
+                    title: 'Retries',
+                    width: 0,
+                    render: (_, [, , retries]) => {
+                        return <RetryStatusIcon retries={retries} />
+                    },
+                },
+                {
+                    title: 'Event',
+                    key: 'event',
+                    className: 'max-w-80',
+                    render: (_, [event]) => {
+                        return event.event ? (
+                            <PropertyKeyInfo value={event.event} type={TaxonomicFilterGroupType.Events} />
+                        ) : (
+                            <EmptyColumn />
+                        )
+                    },
+                },
+                {
+                    title: 'Person',
+                    key: 'person',
+                    render: (_, [, person]) => {
+                        return person ? <PersonDisplay person={person} withIcon /> : <EmptyColumn />
+                    },
+                },
+                {
+                    title: 'URL / Screen',
+                    key: 'url',
+                    className: 'max-w-80',
+                    render: (_, [event]) =>
+                        event.properties['$current_url'] || event.properties['$screen_name'] ? (
+                            <span>{event.properties['$current_url'] || event.properties['$screen_name']}</span>
+                        ) : (
+                            <EmptyColumn />
+                        ),
+                },
+                {
+                    title: 'Library',
+                    key: 'library',
+                    className: 'max-w-80',
+                    render: (_, [event]) => {
+                        return event.properties['$lib'] ? <span>{event.properties['$lib']}</span> : <EmptyColumn />
+                    },
+                },
+                {
+                    title: 'Time',
+                    key: 'time',
+                    className: 'max-w-80',
+                    render: (_, [event]) => {
+                        return event.timestamp ? <TZLabel time={event.timestamp} /> : <EmptyColumn />
+                    },
+                },
+                {
+                    key: 'actions',
+                    width: 0,
+                    render: function RenderActions(_, row) {
+                        return (
+                            <div className="flex gap-1">
+                                <RunRetryButton
+                                    loadingRetries={loadingRetries}
+                                    row={row}
+                                    retryHogFunction={retryHogFunction}
+                                />
```

Comment:
I think this should show the success status after you click retry and it succeeds instead of it being in the first column to make it clearer this is ephemeral